### PR TITLE
UI/Feature polish

### PR DESCRIPTION
**Pull request by Firstyear**
Post release some small user issues arose

* Optimise the autofocus for logins with passkeys to limit clicks
* Sort login mechs by strength
* Fix cookies to persist between browser restarts

Fixes #3187 (mostly) and @adamcstephens issues from matrix

Checklist

- [x] This PR contains no AI generated code
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: server/core/templates/login_webauthn.html [hunk 1]
```diff
@@ -16,16 +16,14 @@
...
+        <button hx-disable type="button" autofocus=true class="btn btn-dark autofocus"
             id="start-passkey-button">Use Passkey</button>
     </form>
     (% else %)
     <form id="cred-form" action="/ui/login/seckey" method="POST">
         <input hidden="hidden" name="cred" id="cred">
-    
-        <button type="button" class="btn btn-dark" id="start-seckey-button"
-            >Use Security Key</button>
+        <button hx-disabel type="button" autofocus=true class="btn btn-dark autofocus"
```
Comment (gbw) [inline server/core/templates/login_webauthn.html:25]: typo disable

File: server/core/templates/login_mech_choose.html [hunk 2]
```diff
@@ -11,6 +11,7 @@
 			<form id="login" action="/ui/login/mech_choose" method="post">
 				<input type="hidden" id="mech" name="mech" value="(( mech.value ))" />
 				<button
+					(% if mech.autofocus %)autofocus=true(% endif %)
```
Comment (yaleman) [inline server/core/templates/login_mech_choose.html:14]: This doesn't need the `=true` [per the usual ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus)

File: server/core/templates/login_webauthn.html [hunk 3]
```diff
@@ -16,16 +16,14 @@
     (% if passkey %)
     <form id="cred-form" action="/ui/login/passkey" method="POST">
         <input hidden="hidden" name="cred" id="cred">
-    
-        <button hx-disable type="button" class="btn btn-dark"
+        <button hx-disable type="button" autofocus=true class="btn btn-dark autofocus"
```
Comment (yaleman) [inline server/core/templates/login_webauthn.html:19]: You've got autofocus listed twice?

File: server/core/templates/login_webauthn.html [hunk 4]
```diff
@@ -16,16 +16,14 @@
...
             id="start-passkey-button">Use Passkey</button>
     </form>
     (% else %)
     <form id="cred-form" action="/ui/login/seckey" method="POST">
         <input hidden="hidden" name="cred" id="cred">
-    
-        <button type="button" class="btn btn-dark" id="start-seckey-button"
-            >Use Security Key</button>
+        <button hx-disable type="button" autofocus=true class="btn btn-dark autofocus"
```
Comment (yaleman) [inline server/core/templates/login_webauthn.html:25]: Also here

**Review by yaleman — CHANGES_REQUESTED** [review]

**Review by yaleman — APPROVED** [review]

**Pull request merged**